path_handler.py

```diff
@@ -0,0 +1,15 @@
+import os
```

Comment:
What's your suggestion? I realize we probably shouldn't have it. But it makes pathing much easier. We could define global variables. Haha